harness.sh

```diff
@@ -224,6 +224,7 @@ fi
 
 if [ "${NETWORK}" = "simnet" ]; then
 tmux send-keys "dcrd -C ${DCRD_CONF} \
+--appdata=${NODES_ROOT}/master \
```

Comment:
Should probably set the log dir as well?

Reply:
The default log directory is derived of the appdata set.